### PR TITLE
Add fields to 2 queries.

### PR DESCRIPTION
**Pull request by AngusCollett**
Adding fields to the result set of `sp_retrieve_sample.sql` and `sp_retrieve_dc.sql` to ensure compatibility with the corresponding dna.xml objects in GDA.

### CODE REVIEW

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql [hunk 1]
```diff
@@ -13,9 +13,11 @@ BEGIN
 					dc.detectorId "detectorId",
 					dc.blSubSampleId "blSubSampleId",
 					dc.dataCollectionNumber "dcNumber",
+					dc.APERTUREID "apertureId",
+					dc.POSITIONID "positionId",
 					dc.startTime "startTime",
 					dc.endTime "endTime",
-					dc.runStatus "status",
+					dc.runStatus "runStatus",
```
Comment (KarlLevik) [inline schemas/ispyb/stored_programs/sp_retrieve_dc.sql:20]: Afraid we can't change the alias for a column in an existing procedure as it's already in use (e.g. by the Python API). I suggest you instead add the column again in the SELECT, but with the "runStatus" alias.

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql [hunk 2]
```diff
@@ -71,13 +74,16 @@ BEGIN
...
+					dc.dataCollectionGroupId "groupId",
 					dc.detectorId "detectorId",
 					dc.blSubSampleId "blSubSampleId",
 					dc.dataCollectionNumber "dcNumber",
+					dc.APERTUREID "apertureId",
+					dc.POSITIONID "positionId",
 					dc.startTime "startTime",
 					dc.endTime "endTime",
-					dc.runStatus "status",
+					dc.runStatus "runStatus",
```
Comment (KarlLevik) [inline schemas/ispyb/stored_programs/sp_retrieve_dc.sql:86]: Same comment as above.

**Review by KarlLevik — CHANGES_REQUESTED** [review]
Looks OK except the renamed column alias in `retrieve_dc`, see comments.

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql (hunk 1 above)
Comment (AngusCollett) [reply schemas/ispyb/stored_programs/sp_retrieve_dc.sql:20]: Sure, I'll just change it on the Java side. It doesn't have to be called runStatus I just like to keep things on the descriptive side.

**Review by KarlLevik — APPROVED** [review]

**Pull request merged**